### PR TITLE
Updating gorm and postgres driver

### PR DESCRIPTION
**Pull request by tahmidefaz**
## What?
PR to update to the latest gorm and postgres driver and it's dependencies.

## Why?
This will allow us to use gorm `Automigrate` to add `unique` constraint to columns.

## How?
* updated to the latest gorm and postgres driver functions
*  ran `go mod tidy`

## Testing
Not applicable.

## Anything Else?
Any other notes about the PR that would be useful for the reviewer. 

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,22 +3,18 @@ module github.com/redhatinsights/payload-tracker-go
...
 	github.com/google/uuid v1.3.0
+	github.com/kr/pretty v0.2.1 // indirect
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.16.0
 	github.com/prometheus/client_golang v1.11.0
-	github.com/qri-io/jsonschema v0.2.1
 	github.com/redhatinsights/app-common-go v1.5.1
 	github.com/redhatinsights/platform-go-middlewares v0.9.0
-	github.com/redis-go/redcon v0.2.0 // indirect
-	github.com/redis-go/redis v0.0.0-20190129163758-e7bb29b08926
```
Comment (aleccohan) [inline go.mod:18]: NICE!  I really like the `go mod tidy`!
I have to remember to use that whenever I push code up!

**Review by aleccohan — APPROVED** [review]
Looks like tests pass!  looks great to me

File: go.mod (hunk 1 above)
Comment (tahmidefaz) [reply go.mod:18]: Yes, it cleans up unused libraries :100:

Comment (tahmidefaz) [reply go.mod:18]: Yes, it cleans up unused libraries :100:

**Comment by SteveHNH** [comment]
sweet! Looking forward to merging this one Monday!

**Pull request merged**